### ISSUE TITLE
When training ,i can not get some information for example mAP 、T 、P 

### ISSUE THREAD

**Issue by litingting1111** [ultralytics/yolov5#2445]
I get right labels and images,but i get this,i do not know why

![image](https://user-images.githubusercontent.com/80535260/110963779-74509e80-838d-11eb-9244-c4a80d4f293e.png)

**Comment by litingting1111** [comment]
my picture  size is 720 ×1160 ，and my  model size is 640 × 640 ，i do not how to get mAP、T、P

**Comment by glenn-jocher** [comment]
@litingting1111 you have no labels in your validation set, this is what targets=0 means. You have not set up your dataset correctly, so it can not find your labels. Please follow the directions in the Train Custom Data Tutorial to get started:

## YOLOv5 Tutorials

* [Train Custom Data](https://docs.ultralytics.com/yolov5/tutorials/train_custom_data)&nbsp; 🚀 RECOMMENDED
* [Weights & Biases Logging](https://github.com/ultralytics/yolov5/issues/1289)&nbsp; 🌟 NEW
* [Multi-GPU Training](https://docs.ultralytics.com/yolov5/tutorials/multi_gpu_training)
* [PyTorch Hub](https://docs.ultralytics.com/yolov5/tutorials/pytorch_hub_model_loading)&nbsp; ⭐ NEW
* [ONNX and TorchScript Export](https://docs.ultralytics.com/yolov5/tutorials/model_export)
* [Test-Time Augmentation (TTA)](https://docs.ultralytics.com/yolov5/tutorials/test_time_augmentation)
* [Model Ensembling](https://docs.ultralytics.com/yolov5/tutorials/model_ensembling)
* [Model Pruning/Sparsity](https://docs.ultralytics.com/yolov5/tutorials/model_pruning_and_sparsity)
* [Hyperparameter Evolution](https://docs.ultralytics.com/yolov5/tutorials/hyperparameter_evolution)
* [Transfer Learning with Frozen Layers](https://docs.ultralytics.com/yolov5/tutorials/transfer_learning_with_frozen_layers)&nbsp; ⭐ NEW
* [TensorRT Deployment](https://github.com/wang-xinyu/tensorrtx)
